src/DynamoCore/Graph/ModelBase.cs

```diff
@@ -13,7 +13,7 @@ namespace Dynamo.Graph
     /// <summary>
     /// SaveContext represents several contexts, in which node can be serialized/deserialized.
     /// </summary>
-    public enum SaveContext { File, Copy, Undo, Preset, None };
+    public enum SaveContext { [Obsolete("Use Save or SaveAs, instead of File")] File, Copy, Undo, Preset, None, Save, SaveAs };
```

Comment:
I believe this is safe as long as the new enums are only added to the end of the enum. The reason is because these are simply compiled to `int`s in increasing order. (as long as the ints are not specified - I think that is actually even safer)